### PR TITLE
DO NOT MERGE YET - Introducing the DataStax driver

### PR DESCRIPTION
**Pull request by michaelarusso**
- Introduce the DataStax driver as a new cluster implementation
- Re-write map serialization to use CQL and the DataStax driver instead of Astyanax
- Standardize tests to use ideal cassandra settings

### CODE REVIEW

File: stack/corepersistence/common/src/main/java/org/apache/usergrid/persistence/core/datastax/TableDefinition.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
+/*
+ *
+ *  * Licensed to the Apache Software Foundation (ASF) under one
```
Comment (johnament) [inline stack/corepersistence/common/src/main/java/org/apache/usergrid/persistence/core/datastax/TableDefinition.java]: Please reformat the `* *` style headers into a single `*`

Comment (michaelarusso) [inline stack/corepersistence/common/src/main/java/org/apache/usergrid/persistence/core/datastax/TableDefinition.java]: good catch.  I'll fix that now.

**Comment by michaelarusso** [comment]
```[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Usergrid Parent .................................... SUCCESS [  2.458 s]
[INFO] Usergrid Build Tools ............................... SUCCESS [  0.310 s]
[INFO] Usergrid Config .................................... SUCCESS [  0.108 s]
[INFO] Usergrid Persistence ............................... SUCCESS [  0.133 s]
[INFO] Usergrid Model ..................................... SUCCESS [  1.740 s]
[INFO] Usergrid Common .................................... SUCCESS [  9.293 s]
[INFO] Usergrid Collection ................................ SUCCESS [ 44.376 s]
[INFO] Usergrid Map ....................................... SUCCESS [  8.454 s]
[INFO] Usergrid Queue ..................................... SUCCESS [  2.132 s]
[INFO] Usergrid Queryindex ................................ SUCCESS [ 11.088 s]
[INFO] Usergrid Test Utils ................................ SUCCESS [  1.842 s]
[INFO] Usergrid Graph ..................................... SUCCESS [01:22 min]
[INFO] Usergrid Cache ..................................... SUCCESS [  6.024 s]
[INFO] Usergrid Core ...................................... SUCCESS [01:42 min]
[INFO] Usergrid Services .................................. SUCCESS [08:01 min]
[INFO] Usergrid REST ...................................... SUCCESS [08:40 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 21:14 min
[INFO] Finished at: 2016-02-13T10:45:24-08:00
[INFO] Final Memory: 63M/447M
[INFO] ------------------------------------------------------------------------```

**Comment by michaelarusso** [comment]
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 22:18 min
[INFO] Finished at: 2016-02-14T16:43:18-08:00
[INFO] Final Memory: 62M/446M
[INFO] ------------------------------------------------------------------------
```

**Pull request closed without merging**